LineSensorBot/src/main/java/frc/robot/Robot.java

```diff
@@ -7,6 +7,8 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.DigitalInput;
```

Comment:
Is this import used?

Reply:
DigitalInput will probably be used instead of AnalogInput. We only use AnalogInput here because we plugged the line sensor into Analog. It should be plugged into Digital in the real bot, though.